src/k8s/cmd/k8s/node_status.go

```diff
@@ -0,0 +1,40 @@
+package k8s
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/spf13/cobra"
+
+	apiv1 "github.com/canonical/k8s/api/v1"
+	cmdutil "github.com/canonical/k8s/cmd/util"
+	"github.com/canonical/k8s/pkg/client/k8sd"
+
+	"github.com/canonical/lxd/shared/api"
+)
+
+// GetNodeStatus retrieves the NodeStatus from k8sd client.
```

Comment:
nit: it retrieves it from the server - the client just does the underlying http request.

Also, let's not overdo it with the line breaks. The comments can be at least the same length as the code.